src/client/containers/RepoChangelog/index.js

```diff
@@ -0,0 +1,43 @@
+/* flow */
+
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { getChangelog } from 'ducks/repositories/selectors'
+import ChangesetList from 'components/ChangesetList'
+
+export type Props = {
+  params: Object,
+  branch: string,
+  repoName: string,
+  commits: Array<any>
+};
+
+class RepoChangelog extends Component {
+  // shouldComponentUpdate(nextProps, nextState) {
+  //   return nextProps.commits
+  // }
+
+  onSelectedChangesetsChange = (e, selectedChangesNumber) => {
+    this.setState({ enabled: selectedChangesNumber in [2, 1] })
```

Comment:
Means that we are able to select 1 or 2 changesets, but no more, or basically `selectedChangesNumber === 1 or selectedChangesNumber === 2`